build.gradle.kts

```diff
@@ -32,6 +33,7 @@ plugins {
   id("eclipse")
   id("polaris-root")
   alias(libs.plugins.rat)
+  alias(libs.plugins.jandex) apply false
```

Comment:
It was suggested as a workaround for this bug in the Jandex plugin:

https://github.com/kordamp/jandex-gradle-plugin/issues/25

I will put a note so that we don't forget.